src/podium/deck.py

```diff
@@ -78,6 +79,13 @@ def __init__(self, filename, app):
         self.reversed_displays = False
         self.paused = False
 
+    @property
+    def resource_path(self):
+        try:
+            return os.path.join(self.app._impl.resource_path, 'app')
+        except Exception:
+            return os.path.sep.join(os.path.dirname(os.path.abspath(podium.__file__)).split(os.path.sep)[:-2])
```

Comment:
This is not self-explanatory, unfortunately. It has at least two issues:

1. The exception is too broad. -- _What is actually happening when something goes wrong? An `AttributeError` on `resource_path`?_
2. Line 87 does too many things. -- _Resources are probably located two levels below the installation location of a package. Can we put this in variable names?_